msg/ImageWithRegionOfInterest.msg

```diff
@@ -4,4 +4,9 @@ std_msgs/Header header
 
 sensor_msgs/Image image
 
-sensor_msgs/RegionOfInterest regionOfInterest
\ No newline at end of file
+sensor_msgs/RegionOfInterest regionOfInterest
+
+uint16 original_width
+
+uint16 original_height
+
```

Comment:
It's strange to send a region of interest if the image is already only the region of interest. If the image is the whole image, original width and height are not needed.

Reply:
I wanted to keep it for the transformer, but the transformer has to know the size of the input image anyway, right? So I think we can scrap the originl image size.